### ISSUE TITLE
Add asset/icon management

### ISSUE THREAD

**Issue by mlynch** [ionic-team/trapeze#104]
I think the work I did on @ionic-team/capacitor-assets should be merged into Trapeze instead since the tasks of managing assets is in scope for Trapeze. https://github.com/ionic-team/capacitor-assets/issues/291

**Comment by michaelkariv** [comment]
I agree. I came here looking for an example of the configuration snippet that would copy a generated launcher icons set to the android configuration. Kind of like 
`      - path: drawable
        file: icon.png
        source: ./public/assets/icon/icon.png
`
but instead of one file there would be a folder of drawables generated by [Android Asset Studio](https://romannurik.github.io/AndroidAssetStudio/index.html)

If my undertsanding of the flow you suggest is correct, then trapeze would have a command to take the same SVG file that I fed into Android Asset Studio, and extra parameters for the colors, and styling enum and a bit more, and generate ic_launcher drawables, but instead of downloading them, would place it in the android/app/src/main/assets direclty.

That would be a good shortcut.

I however would still appreciate a configuration exmaple for copying the whole folder. It is because sometimes one want to interfere in the process of generating the drawables, maybe using other tools, e.g. multiple colors.

**Comment by mlynch** [comment]
This is what capacitor-assets does, though it's a bit more opinionated for Capacitor projects, but it might work fine for the broader set of projects that Trapeze supports https://github.com/ionic-team/capacitor-assets

**Comment by robingenz** [comment]
We are not going to merge asset generation into trapeze — `@capacitor/assets` still covers icon/splash generation and is a better home for it. If you only need to place an already-generated folder, `android.copy` / `project.copy` accept directories as `src`, so you can copy a whole drawable set per build target. Closing this; happy to reopen if there is a concrete gap that `@capacitor/assets` plus a copy operation cannot cover.
